### ISSUE TITLE
Just a ConvNet 

### ISSUE THREAD

**Issue by gabrielleyr** [deepmind/learning-to-learn#16]
If I run 
$ python evaluate.py --optimizer=Adam --problem=cifar 
am I training the ConvNet for 100 epochs in batches of 128? I'm trying to use this as solely an example of using Sonnet to create a ConvNet, without any meta-learning. 

I'm confused because when I delete the get_default_net_config function in util.py, which only has information about the metalearning net, and run the command above, there's an error. Why is the net config for the coordinatewise deep LSTM necessary for just training the ConvNet with Adam?

**Comment by sergomezcol** [comment]
Yes, exactly. That will just train a convnet on CIFAR for 100 epochs using tf.train.AdamOptimizer.

You can see how this happens in `evaluate.py`, lines 53-65. We always call `get_config` in the same way, but in the Adam case we don't do anything with `net_config` and `net_assignments`. We should have probably written this in a different way, now that I think about it. But that's the reason why it crashes when you delete `get_default_net_config`.

You can see how the Sonnet network is built in `problems.cifar10()`. The default parameters (from `util.py`) are `conv_channels=(16, 16, 16)` and `linear_layers=(32,)`. You can probably ignore all the dataset/queue-related stuff.

**Comment by gabrielleyr** [comment]
I'm trying to add a method of checking train/val accuracy during training for just the ConvNet. It seems like the network is built only once with all the image batches and label batches, but I want to access the network or at least its weights to calculate the loss and then accuracy on the entire training set and entire validation set each epoch. Do you have any suggestions for doing so? 

It would most likely look like a call of some check_accuracy function once an epoch within evalulate.py's main function. Does it make sense to make separate sessions for checking training accuracy and validation accuracy, and call run_epoch on each session, leaving out the [update] argument? How can they access the weights?
